views/default/panels/db/detail.php

```diff
@@ -98,7 +98,7 @@
 function debug_db_detail() {
     $('.db-explain a').on('click', function(e) {
         e.preventDefault();
-        
+
```

Comment:
It's automatic PhpStorm's removal of whitespace on blank lines. I agree that it should not be in this commit...